### PR TITLE
Project crud

### PR DESCRIPTION
**Pull request by vladfreel**
Projects CRUD

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -2,4 +2,10 @@ class Project < ApplicationRecord
   has_one :image, as: :imageable
   has_many :resource_skills, as: :skillable
   has_many :skills, through: :resource_skills
+  validates :name, presence: true
+
+  def self.get_skills(id)
```
Comment (drkmen) [inline app/models/project.rb]: really? you have relation...

**Comment by drkmen** [comment]
We don't need index and show actions. Remove them, all related stuff to them and all empty files. .scss, .coffee, _spec, _helper etc.

File: app/views/projects/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+- @projects.each do |project|
+  = project.name
+  = project.description
+  = project.name
```
Comment (Tivgres) [inline app/views/projects/index.html.haml]: Name here is twice

File: app/views/projects/show.html.haml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+= @project.name
+= @project.description
+= @project.name
```
Comment (Tivgres) [inline app/views/projects/show.html.haml]: Name here is twice too

File: spec/controllers/projects_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe ProjectsController, type: :controller do
+  before do
+    @employee = FactoryBot.create(:employee, :admin_full)
```
Comment (Tivgres) [inline spec/controllers/projects_controller_spec.rb]: better to use a let method

**Pull request merged**